packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx

```diff
@@ -188,24 +189,35 @@ const UnstakePositionModal: React.FC<Props> = ({
           <div className="box-item">
             <div className="item-content">
               <div>
-                <div className="label-large">Total Amount</div>
+                <div className="label-large">
+                  {t("UnstakePosition:confStakeModal.totalAmt")}
+                </div>
                 <div className="value-large">{totalLiquidityUSD}</div>
               </div>
             </div>
           </div>
           <WarningCard
-            title={"Important Note"}
+            title={t("UnstakePosition:confStakeModal.warning.title")}
             icon={<IconCircleExclamationMark />}
             content={
               <UnstakeWarningContentWrapper>
-                Your APR will reduce from {currentPercent} →{" "}
-                <span className="unstake-percent">{swapFeePercent}</span>
+                <Trans
+                  ns="UnstakePosition"
+                  i18nKey="confStakeModal.warning.content"
+                  values={{
+                    currentPercent,
+                    swapFeePercent,
+                  }}
+                >
+                  Your APR will reduce from {currentPercent} →{" "}
+                  <span className="unstake-percent">{swapFeePercent}</span>
```

Comment:
it maybe redundant string

Reply:
This is not redundant, Trans component treats it like a string template